app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
Sorry, I didn't explain myself better.  Display of "VM" is not new to manageiq.  All of the other UI would have had to either solve this problem via the message catalog or something else, or kept it as "Vm" for display.  I'm suggesting that all places that want to display Vm as VM would want to do this.  I don't know, it just seemed odd if this is the only place we do it.